src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -18,9 +18,9 @@
 #include <toolkits/style_transfer/style_transfer_model_definition.hpp>
 #include <toolkits/util/training_utils.hpp>
 
-#ifdef __APPLE__
+#ifdef HAS_MPS
```

Comment:
Out of curiosity, where is `HAS_MPS` defined?

Reply:
Top Level CMakeLists.txt